db/migrate/20200301015408_add_assocation_between_assignments_and_users.rb

```diff
@@ -0,0 +1,9 @@
+class AddAssocationBetweenAssignmentsAndUsers < ActiveRecord::Migration[6.0]
+  def change
+    add_reference :assignments, :assigned_by, references: :staffs, index: true
+    add_foreign_key :assignments, :staffs, column: :assigned_by_id
```

Comment:
i believe that if you add "add_reference" you don't need to put add_foreign_key to the staffs table if you include the association in the model? same comment with assignments